packages/bruno-lang/v2/src/bruToJson.js

```diff
@@ -57,13 +57,14 @@ const grammar = ohm.grammar(`Bru {
   // Dictionary Blocks
   dictionary = st* "{" pairlist? tagend
   pairlist = optionalnl* pair (~tagend stnl* pair)* (~tagend space)*
-  pair = st* (quoted_key | key) st* ":" st* value st*
+  pair = st* (multiline_key | quoted_key | key) st* ":" st* value st*
   disable_char = "~"
   quote_char = "\\""
   esc_char = "\\\\"
   esc_quote_char = esc_char quote_char
   quoted_key_char = ~(quote_char | esc_quote_char | nl) any
   quoted_key = disable_char? quote_char (esc_quote_char | quoted_key_char)* quote_char
+  multiline_key = disable_char? multilinetextblock
```

Comment:
@sid-bruno @pooja-bruno @bijin-bruno 

We cannot and should not support multiline keys. Yaml does not support it and this will be a blocker for people to migrate over to yaml. Lets discus this in person tomorrow.